digitaltwins/samples/service/invoke_command.js

```diff
@@ -23,7 +23,7 @@ async function main() {
 
     // Invoke a command
     console.log('invoking command ' + commandName + ' on component instance' + componentInstanceName + ' for device ' + deviceId + '...');
-    const invokeComponentCommandRespone = await digitalTwinServiceClient.invokeCommand(deviceId, componentInstanceName, commandName, commandArgument);
+    const invokeComponentCommandRespone = await digitalTwinServiceClient.invokeComponentCommand(deviceId, componentInstanceName, commandName, commandArgument);
```

Comment:
>digitalTwinServiceClient.invokeComponentCommand [](start = 48, length = 47)

The uses the wrong path.  Needs to use the components rather than interfaces.